python/smashbox/utilities/__init__.py

```diff
@@ -477,6 +477,9 @@ def reset_server_log_file():
         log file for the test run
     """
 
+    if not config.copy_server_log:
```

Comment:
Using a "copy_server_log" flag to prevent deleting the log..... I find it confusing

Do we want to copy the log but not delete it? Delete it but not copy?